docs/modules/ROOT/pages/how-tos/configuring/creating-secrets.adoc

```diff
@@ -2,18 +2,23 @@
 
 When you build your pipeline, you might want to add tasks that require **secrets** in order to access external resources.
 
-NOTE: One such task is the link:https://github.com/konflux-ci/build-definitions/tree/main/task/sast-snyk-check[sast-snyk-check] task that uses the third-party service link:https://snyk.io/[snyk] to perform static application security testing (SAST) as a part of the default {ProductName} pipeline. Use this procedure to upload your snyk.io token. Name the secret `sast_snyk_task` so that the snyk task in the {ProductName} pipeline will recognize it and use it.
+.Secrets fall into three catagories of when they can be added:
+
+. Before a component is added. If a secret is needed to access the source control platform (like xref:creating-secrets.adoc#creating-source-control-secrets[GitLab]), then the secret must be created before the component is created.
```

Comment:
```suggestion
. Before a component is added. If a secret is needed to access the source control platform (like xref:creating-secrets.adoc#creating-source-control-secrets[GitLab]), you must create a secret before you create the component.
```

The general rule of thumb for RH docs is to be as user-oriented as we can. Meaning we put 'you' everywhere to emphasize this is an action a user needs to do. But then it's not a requirement.